### DISCUSSION TITLE
Required fields only recognisable as required when empty - sufficient for SC 3.3.2?

### DISCUSSION THREAD

**Discussion by JAWS-test** [w3c/wcag#3852] [Q&A]
I have an application in which required fields are not marked, but empty required fields are already displayed initially with an error message ("Please fill required field"), which is also linked to the field correctly. As soon as the required field is filled, the error message disappears and the field is no longer recognizable as a required field. Does this mean that SC 3.3.2 is fulfilled? I would say: Yes. What do others think?

**Comment by fstrr** [comment]
This issue is labelled as a discussion, so we’re moving this to Discussions. There doesn’t seem to be an update to make to the documentation, but if that changes, we can move it back to the issues list.
